src/patternfly/components/Dropdown/examples/Dropdown.md

```diff
@@ -161,7 +179,8 @@ The dropdown menu can contain either links or buttons, depending on the expected
 | `.pf-c-dropdown__toggle-icon` | `<i>` | Defines the dropdown toggle icon. |
 | `.pf-c-dropdown__toggle-text` | `<span>` | Defines the dropdown toggle text. **Required when text is present, adds truncation**. |
 | `.pf-c-dropdown__toggle-check` | `<label>` | Defines a checkbox in the toggle area of a split button dropdown. |
-| `.pf-c-dropdown__toggle-button` | `<button>` | Defines a button in the toggle area of a split button dropdown. |
+| `.pf-c-dropdown__toggle-button` | `<button>` | Defines the toggle button for a split button dropdown. |
+| `.pf-c-dropdown__toggle-action` | `<button>` | Defines an action in the toggle area of a split button dropdown. |
```

Comment:
Looks like this isn't being used.

Reply:
Oops! Artifact from a different approach.